server/index.js

```diff
@@ -44,7 +44,7 @@ server.start = function(port, callback){
   console.log(`http://localhost:${port}/`)
   server.listen(port, callback)
 }
-
+//add setinterval for purging overdue Prrrs here? ask Jared
```

Comment:
Forgot to remove this comment, this issue is actually part of a different card/issue/PR/branch. Will remove comment.